pkg/config/assets/another_service_config.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
same comment as below. this file I think is leftover from merge or rebase. we should delete this.